react-native/react/tracker/proofs.render.desktop.js

```diff
@@ -1,77 +1,106 @@
 'use strict'
+/* @flow */
 
+// $FlowIssue base-react
 import React, {Component} from '../base-react'
 
-// TODO const when integrating
-const verified = 'verified'
-const checking = 'checking'
-const deleted = 'deleted'
-const unreachable = 'unreachable'
-const pending = 'pending'
+import {identify} from '../keybase_v1'
+
+import type {RemoteProof, LinkCheckResult, ProofState, ProofStatus} from '../constants/types/flow-types'
+
+export type ProofsAndChecks = Array<[RemoteProof, ?LinkCheckResult]>
+export type ProofsProps = {
+  proofsAndChecks: ProofsAndChecks
+}
 
 export default class ProofsRender extends Component {
-  openLink (proof, platform) {
-    window.open(platform ? proof.platformLink : proof.proofLink)
+  props: ProofsProps;
+
+  // TODO hook this up
+  openLink (url: string): void {
+    window.open(url)
   }
 
-  metaColor (pp) {
-    return {
-      'new': 'orange',
-      deleted: 'red',
-      unreachable: 'red',
-      pending: 'gray'
-    }[pp.proof.meta]
+  metaColor (lcr: ?LinkCheckResult): string {
```

Comment:
I think most of this kind of logic should move up a level , when we make mobile renders we don't also want that logic in the render component